public/manifest.json

```diff
@@ -7,11 +7,6 @@
       "sizes": "64x64 32x32 24x24 16x16",
       "type": "image/x-icon"
     },
-    {
```

Comment:
You can read about manifest.json file here: https://developer.mozilla.org/en-US/docs/Web/Manifest

It's added to CRA boilerplate in order to support PWA stuff:

> A web application manifest, as defined in the [Web Application Manifest](https://w3c.github.io/manifest/) specification, provides information about a web application in a [JSON](https://developer.mozilla.org/en-US/docs/Glossary/JSON) text file, necessary for the web app to be downloaded and be presented to the user similarly to a native app (e.g., be installed on the homescreen of a device, providing users with quicker access and a richer experience). PWA manifests include its name, author, icon(s), version, description, and list of all the necessary resources (among other things).